pyspark/bigdl/keras/converter.py

```diff
@@ -1190,7 +1197,7 @@ def create_atrousconvolution1d(self):
         if not self.config["bias"]:
             raise Exception("Only bias=True is supported for AtrousConvolution1D")
 
-        h = self.input_shape[2]
+        h = int(self.input_shape[1])
```

Comment:
add a dedicated unitest for this which would fail if we use `self.input_shape[2]`